ReactAndroid/src/main/java/com/facebook/react/shell/MainReactPackage.java

```diff
@@ -45,12 +47,18 @@
  */
 public class MainReactPackage implements ReactPackage {
 
+  private Context mActivityContext;
+
+  public MainReactPackage(Context activityContext) {
+    mActivityContext = activityContext;
+  }
```

Comment:
I have a diff out to open source the Dialog module which does something similar. Should land it soon-ish.